src/components/translator/fr.json

```diff
@@ -0,0 +1,16 @@
+{
+    "finalSchema": "Schéma Final",
+    "formIntent": "Intention du Formulaire",
+    "formRender": "Rendu du Formulaire",
+    "formRenderMenuLink": "Rendu du Formulaire",
+    "formRenderTestTitle": "Test de Rendu du Formulaire",
+    "generatedForm": "Formulaire Généré",
```

Comment:
Some of these keys are are not essential to the form engine but rather other esms, in this case `Form Render` we so some clean up of this